setupext.py

```diff
@@ -68,15 +69,8 @@ def check_for_openmp():
     tmp_dir = tempfile.mkdtemp()
     start_dir = os.path.abspath('.')
 
-    if os.name == 'nt':
-        # TODO: make this work with mingw
-        # AFAICS there's no easy way to get the compiler distutils
-        # will be using until compilation actually happens
-        compile_flag = '-openmp'
-        link_flag = ''
-    else:
-        compile_flag = '-fopenmp'
-        link_flag = '-fopenmp'
+    compile_flag = '-fopenmp'
+    link_flag = '-fopenmp'
```

Comment:
Why was this if statement deleted?

Reply:
Because of line 62 above. 
```
if sys.version_info[:3] == (3, 5, 0) or os.name == 'nt':
``
This statement can never be true as the function already return (if it'd be true).

Reply:
Oh I see you mentioned it. It's true on Windows:

https://stackoverflow.com/questions/1325581/how-do-i-check-if-im-running-on-windows-in-python

Can you please restore the if statement?

Reply:
Oh, huh, good point. That's a mistake! We definitely should support OpenMP on Windows.

Can you please revert changing the if statement and then we'll fix the other check in a separate PR.